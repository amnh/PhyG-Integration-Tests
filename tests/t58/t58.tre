(Afrikaans:4087.15,(Manx:273.15,(OldPrussian:791.15,((Cornish:747.6,(Danish:464.15,Dutch:641.65)HTU78:453.65)HTU81:381.45000000000005,(Breton:867.8,(Welsh:1123.15,((OldIrish:591.05,(Irish:236.75,ScottishGaelic:197.05)HTU69:358.15)HTU79:1003.65,(((ArmenianEastern:1631.9499999999998,TocharianB:1582.1)HTU75:1077.15,(AlbanianTosk:1304.65,TocharianA:952.05)HTU88:992.85)HTU90:2248.7000000000003,((EasternFrisianSaterland:910.55,(Faroese:570.15,Scots:827.15)HTU80:519.05)HTU86:2829.65,((German:1200.85,WesternFrisian:1927.35)HTU93:3180.15,(((ArmenianWestern:661.8499999999999,ClassicalArmenian:782.15)HTU72:711.85,(Latvian:634.95,Lithuanian:1106.1)HTU74:731.65)HTU89:3639.65,((Latgalian:960.35,Samogitian:1454.35)HTU77:4115.15,((EnglishAmerican:995.85,(EnglishBritish:131.7,PennsylvaniaDutch:80.25)HTU59:790.35)HTU70:4839.25,(NorthFrisianOomrang:11401.1,(((Swedish:2698.25,((Icelandic:1067.15,(LowSaxon:1096.85,OldHighGerman:734.0)HTU73:826.15)HTU51:2039.3000000000002,((KolschColognian:1801.5,Norweigan:664.85)HTU52:523.15,(Luxemburgish:903.15,SchwyzerdutschSwissGerman:1423.35)HTU53:589.25)HTU54:1952.15)HTU56:2529.75)HTU62:2232.15,(AncientGreek:2544.65,(Gothic:2560.05,(CentralFranconianRipuarian:1091.15,Yiddish:1227.4499999999998)HTU60:1106.65)HTU63:1153.65)HTU64:2076.75)HTU66:3401.65,(Luwian:3494.0,((Assamese:1010.2,(Avestan:473.65,SanskritVedic:583.75)HTU55:731.15)HTU57:2301.15,((AncientIonicGreek:1236.65,PharasaGreek:1937.55)HTU58:1753.9499999999998,(GreekModern:4099.15,SouthernTsakonian:2455.55)HTU61:1184.15)HTU65:1308.15)HTU67:1324.25)HTU68:3428.65)HTU71:7176.85)HTU76:6195.65)HTU91:3262.65)HTU92:2568.4)HTU94:2476.85)HTU96:2041.65)HTU97:1927.35)HTU95:2074.0499999999997)HTU87:1064.4)HTU85:539.65)HTU84:477.65)HTU83:472.55)HTU82:458.65)HTU98:3891.65)HTU50[81600.8];
