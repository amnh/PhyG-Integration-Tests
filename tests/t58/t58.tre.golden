(Afrikaans:4081.15,(Manx:287.15,(OldPrussian:794.15,((Cornish:759.6,(Danish:473.15,Dutch:651.65)HTU78:448.65)HTU81:377.45000000000005,(Breton:872.8,(Welsh:1124.15,((OldIrish:602.05,(Irish:248.75,ScottishGaelic:209.05)HTU69:355.15)HTU79:996.15,(((ArmenianEastern:1632.9499999999998,TocharianB:1585.1)HTU75:1059.65,(AlbanianTosk:1297.15,TocharianA:947.55)HTU88:971.85)HTU90:2222.7000000000003,((EasternFrisianSaterland:919.55,(Faroese:579.15,Scots:835.15)HTU80:514.05)HTU86:2819.65,((German:1208.35,WesternFrisian:1921.35)HTU93:3145.65,(((ArmenianWestern:675.8499999999999,ClassicalArmenian:801.15)HTU72:690.35,(Latvian:651.95,Lithuanian:1118.1)HTU74:715.15)HTU89:3613.65,((Latgalian:978.35,Samogitian:1464.35)HTU77:4082.15,((EnglishAmerican:1029.85,(EnglishBritish:181.7,PennsylvaniaDutch:130.25)HTU59:776.35)HTU70:4785.25,(NorthFrisianOomrang:11400.1,(((Swedish:2738.75,((Icelandic:1140.15,(LowSaxon:1177.85,OldHighGerman:823.5)HTU73:806.15)HTU51:2017.8000000000002,((KolschColognian:1885.5,Norweigan:747.85)HTU52:515.15,(Luxemburgish:989.15,SchwyzerdutschSwissGerman:1504.35)HTU53:584.25)HTU54:1923.15)HTU56:2471.25)HTU62:2162.15,(AncientGreek:2591.65,(Gothic:2617.05,(CentralFranconianRipuarian:1152.15,Yiddish:1271.4499999999998)HTU60:1069.65)HTU63:1094.65)HTU64:2001.75)HTU66:3317.65,(Luwian:3498.5,((Assamese:1093.2,(Avestan:557.65,SanskritVedic:666.75)HTU55:725.15)HTU57:2246.15,((AncientIonicGreek:1310.65,PharasaGreek:2009.55)HTU58:1682.4499999999998,(GreekModern:4167.15,SouthernTsakonian:2515.55)HTU61:1104.65)HTU65:1209.15)HTU67:1244.25)HTU68:3347.15)HTU71:7105.35)HTU76:6123.15)HTU91:3199.65)HTU92:2525.9)HTU94:2441.35)HTU97:2016.65)HTU96:1908.35)HTU95:2055.0499999999997)HTU87:1056.4)HTU85:535.15)HTU84:473.65)HTU83:468.55)HTU82:454.65)HTU98:3878.15)HTU50[81600.8];
